modules/oc-by-example-content.adoc

```diff
@@ -809,41 +809,41 @@ Create an ingress with the specified name
 .Example usage
 [source,bash,options="nowrap"]
 ----
-  # Create a single ingress called 'simple' that directs requests to foo.com/bar to svc
+  # Create a single ingress called 'simple' that directs requests to example.com/bar to svc
```

Comment:
I suggest also changing "bar" here and other similar places. foo/bar is an accepted pair that makes less sense as example/bar.

Reply:
@bergerhoffer I see the same note on auto-generation here as well. Will revert the change. cc @kalexand-rh

Reply:
@bergerhoffer, when you change the CLI code, will you take a look at this, too?